### PR TITLE
Add GMT Applications newspost

### PR DESCRIPTION
**Pull request by osuChaos**
newspost for gmt applications

### CODE REVIEW

File: news/2020-10-04-global-moderation-team-applications-open.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+The team has traditionally added new members through a recommendation process based on observed good behaviour and active presence in-game. While we will continue bringing in new members in this way, we know that not everyone is outspoken about voicing their interest in becoming a part of the team. We want to see who else is out there in the community that has what it takes to join the team.
+
+Please note that we will be considering a few factors along with the answers you submit within the application:
+
+- General requirements to join the GMT have not changed. You can read in detail about what we expect on [the GMT wiki page](https://osu.ppy.sh/help/wiki/People/The_Team/Global_Moderation_Team#becoming-a-global-moderation-team-member), but in summary: you must be active in the in-game chat and/or the forums, you must demonstrate good behaviour by treating fellow community members with respect, you must possess a strong understanding of [osu!'s community rules](https://osu.ppy.sh/help/wiki/Rules) and follow them, and you must already be contributing to bettering the community (by assisting other users, submitting reports for other users breaking the rules, etc).
```
Comment (cl8n) [inline news/2020-10-04-global-moderation-team-applications-open.md:19]:
```suggestion
- General requirements to join the GMT have not changed. You can read in detail about what we expect on [the GMT wiki page](/wiki/People/The_Team/Global_Moderation_Team#becoming-a-global-moderation-team-member), but in summary: you must be active in the in-game chat and/or the forums, you must demonstrate good behaviour by treating fellow community members with respect, you must possess a strong understanding of [osu!'s community rules](/wiki/Rules) and follow them, and you must already be contributing to bettering the community (by assisting other users, submitting reports for other users breaking the rules, etc.).
```

File: news/2020-10-04-global-moderation-team-applications-open.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- Your account age must be older than one year to submit an application. Beyond that, we are ideally looking for people who have been in the community for several years.
+- You must be able to communicate in English as most of the community is English speaking.
+- You must have no major infractions on your account. This means any recent restrictions, multiaccount offenses, extended history of misbehaviour, etc will disqualify you from consideration from the team.
```
Comment (cl8n) [inline news/2020-10-04-global-moderation-team-applications-open.md:22]:
```suggestion
- You must have no major infractions on your account. This means any recent restrictions, multiaccount offenses, extended history of misbehaviour, etc. will disqualify you from consideration from the team.
```

File: news/2020-10-04-global-moderation-team-applications-open.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- You must be willing to handle and moderate inappropriate content that is not suitable for all ages.
+- If you are chosen as a candidate, it is expected that you will be active and behave in a manner befitting a moderator when interacting with the community upon joining the team.
+
+We are not looking to fulfill any particular quota of new team members so any number of qualified applicants may be added. While we are accepting applications from everyone, we are actively seeking new members who are able to help us with these languages: **Arabic, Czech, Dutch, Finnish, Greek, Hebrew, Hungarian, Indonesian, Italian, Malaysian, Romanian, Portuguese, Spanish, Thai, and Vietnamese**
```
Comment (cl8n) [inline news/2020-10-04-global-moderation-team-applications-open.md:26]:
```suggestion
We are not looking to fulfill any particular quota of new team members, so any number of qualified applicants may be added. While we are accepting applications from everyone, we are actively seeking new members who are able to help us with these languages: **Arabic, Czech, Dutch, Finnish, Greek, Hebrew, Hungarian, Indonesian, Italian, Malaysian, Romanian, Portuguese, Spanish, Thai, and Vietnamese**.
```

File: news/2020-10-04-global-moderation-team-applications-open.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+Please be aware that we are expecting a large number of applications so it will take some time for all of the applications to be reviewed. Once we have reviewed all of the applications, you will be contacted by a member of the GMT if we deem you as a suitable addition to the team. We will also endeavour to send out messages to applicants who show promise but we find aren't quite ready yet, in order to try to help them to becoming a candidate to join the team in the future. For those who are accepted, the team currently uses Discord as a primary form of communication and we expect new members to be available on Discord to keep up to date with operations within the team.
+
+Please feel free to drop any questions or concerns in the comments and feel free to reach out to any current GMT member about anything you may want to know. Good luck to everyone applying, and we hope to see some of you with us soon!
+
+—Chaos, Nathanael
```
Comment (cl8n) [inline news/2020-10-04-global-moderation-team-applications-open.md:34]:
```suggestion
—Chaos and Nathanael
```

**Review by cl8n — CHANGES_REQUESTED** [review]

File: news/2020-10-04-global-moderation-team-applications-open.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "Global Moderation Team Applications Open"
+date: 2020-10-04 14:00:00 +0000
+---
+
+Are you interested in helping keep the osu! community clean and suitable for everyone? The Global Moderation Team is now accepting applications for new members!
+
+![](https://assets.ppy.sh/media/gmt_generic.jpg)
```
[the remark this answers is not in the document]
Comment (osuChaos) [reply news/2020-10-04-global-moderation-team-applications-open.md:9]: need @Ephemeralis to change this I believe

[the remark this answers is not in the document]
Comment (Ephemeralis) [reply news/2020-10-04-global-moderation-team-applications-open.md:9]: no harm in leaving it on assets for now

[the remark this answers is not in the document]
Comment (Ephemeralis) [reply news/2020-10-04-global-moderation-team-applications-open.md]: that being said i've moved it across anyway

**Review by cl8n — APPROVED** [review]

**Review by Ephemeralis — APPROVED** [review]

**Pull request merged**